app/governance/components/votingChart/voteGraph.tsx

```diff
@@ -0,0 +1,405 @@
+import React from "react";
+import styles from "./voteGraph.module.scss";
+import Text from "@/components/text";
+import Icon from "@/components/icon/icon";
+import { formatBalance } from "@/utils/formatting";
+import Container from "@/components/container/container";
+
+interface VoteBarGraphProps {
+  yesVotes: number;
+  noVotes: number;
+  abstainVotes: number;
+  vetoVotes: number;
+  size: number;
+}
+
+export const VoteBarGraph = ({
+  yesVotes,
+  noVotes,
+  abstainVotes,
+  vetoVotes,
+  size,
+}: VoteBarGraphProps) => {
+  const totalVotes = yesVotes + noVotes + abstainVotes + vetoVotes;
+
+  const yesPercentage = totalVotes > 0 ? (yesVotes / totalVotes) * 100 : 0;
+  const noPercentage = totalVotes > 0 ? (noVotes / totalVotes) * 100 : 0;
+  const abstainPercentage =
+    totalVotes > 0 ? (abstainVotes / totalVotes) * 100 : 0;
+  const vetoPercentage = totalVotes > 0 ? (vetoVotes / totalVotes) * 100 : 0;
+
+  const yesHeight = (yesPercentage * size) / 150; //150 is to make the bar occupy at max 2/3rd of the total height of the container if an option get 100% votes
+  const noHeight = (noPercentage * size) / 150;
+  const abstainHeight = (abstainPercentage * size) / 150;
+  const vetoHeight = (vetoPercentage * size) / 150;
+
+  return (
+    <div className={styles.statsContainer}>
+      <div className={styles.statsHeader}>
+        <Text font="proto_mono">Voting Stats</Text>
+      </div>
+      <div className={styles.graphContainer}>
+        <div className={styles.barContainer}>
```

Comment:
barContainer can be it's own component instead of repeating same code 4 times. refactor it into a component inside of the same file

Reply:
Yes done this. Also added another component VoteInfo for vote Option Info under the graph bar in the proposal details page